web/vtadmin/src/components/routes/Debug.module.scss

```diff
@@ -56,3 +56,201 @@
 .tabContainer {
     margin: 2.4rem 0;
 }
+
+.danger {
```

Comment:
This unpleasant part of the diff adds a bunch of CSS classes for rendering the colour chips on the debug page. The new JIT compiler in tailwindcss v3 really simply does _not_ work with "runtime classes", e.g., a JavaScript string template like `background-${foo}`.

There are more "elegant" ways to fix this with postcss loops but it looked like it would require adding a new postcss dependency to support loops and... 🤷